Mathlib/SetTheory/Ordinal/FixedPointApproximants.lean

```diff
@@ -131,7 +131,7 @@ theorem lfpApprox_eq_of_mem_fixedPoints {a b : Ordinal} (h_init : x ≤ f x) (h_
     · rw [IH a' ha'b (le_of_not_lt haa), h]
   · exact lfpApprox_monotone f x h_ab
 
-/-- There are distinct ordinals smaller than the successor of the domains cardinals
+/-- There are distinct ordinal indices smaller than the successor of the domains cardinals
```

Comment:
```suggestion
/-- There are distinct indices smaller than the successor of the domain's cardinal yielding the same value
```
It's not the indices that have the same value, but rather the values associated to those indices that are equal.